### ISSUE TITLE
Lolminer not detecting GPU

### ISSUE THREAD

**Issue by GamerWierdo100** [Lolliedieb/lolMiner-releases#2074]
When I try to run LoLminer it gives me this
```
Setup Miner... 
OpenCL driver detected.
Number of OpenCL supported GPUs: 0 
Cuda driver detected.
Number of Cuda supported GPUs: 0 

Connecting to pool... 
DNS over HTTPS resolve succeeded 
Connected to de.ironfish.herominers.com(49.12.87.230):1145  (TLS disabled)
Authorized worker: 406c14bac8709d75045c175bee21a13216107fcb6d5be66e5580a3a1d3a80035.lolMinerWorker 
New extra nonce received: 00
New target received: 000000000232f330
New fishing job with id 0 received 
Start Mining... 
All devices deselected or failed compatiblity check. Closing lolMiner

```
I am running Kubuntu

**Comment by says112** [comment]
How do I fix the same thing in ubuntu 20.04?

**Comment by jgonzis** [comment]
You need a dedicated GPU, it will not work with CPU or iGPU
